doc/geany.txt

```diff
@@ -3042,16 +3039,16 @@ behaviors:
   compiler tab and parse it for errors.
 * Execute commands - are configurable and intended for executing your
   program or other long running programs.  The output is not parsed for errors
-  and is directed to the terminal command selected in preferences.
+  and is directed to the terminal command selected in *Preferences*.
 * Fixed commands - these perform built-in actions:
 
-  * Go to the next error.
-  * Go to the previous error.
-  * Show the build menu commands dialog.
+* Go to the next error.
+* Go to the previous error.
+* Show the build menu commands dialog.
```

Comment:
I think this indentation was intentional for nesting.

Reply:
Yes. BTW the restructuredtext indent should go after the `*`:
```
 *   Go to the next error
```
otherwise it gets shown in a box with green background. But for now I reverted this, it's fixed differently in #2307.

Reply:
It shows correctly here after the revert.

Reply:
Ok, weird. It looks right here too:
https://www.geany.org/manual/dev/index.html#build-menu-configuration

Anyway, I broke up the list in the other pull so that's still valid either way.